docs/RELEASE_NOTES.md

```diff
@@ -27,6 +27,14 @@
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 -->
 
+### 5.1.2 - May 31, 2021
+[Commits](https://github.com/commercetools/commercetools-sync-java/compare/5.1.1...5.1.2) |
+[Javadoc](https://commercetools.github.io/commercetools-sync-java/v/5.1.2/) |
+[Jar](https://bintray.com/commercetools/maven/commercetools-sync-java/5.1.2)
+
+- 🐞 **Bug Fixes** (1)
+  - **Product Sync** - The user is now aware of unresolvable references as the transform service will not skip the products.
+
```

Comment:
How about dependencies update? Can you check which dependencies have been updated version since last release?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/739/commits/5bae2d56979cba90719f566d9ed121876ca6f9db